services/jira/jira-base.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const BaseJsonService = require('../base-json')
+const serverSecrets = require('../../lib/server-secrets')
+
+module.exports = class BaseJiraService extends BaseJsonService {
+  static get category() {
+    return 'issue-tracking'
+  }
+
+  async fetch({ url, options = {}, schema, errorMessages }) {
```

Comment:
It might be better to accept `qs` here rather than the full `options` object. It conveys the intent better.

Reply:
Will do. I can't see any use cases when a service class would need to define anything other than `qs` anyway